src/state/models/ui/create-account.ts

```diff
@@ -125,7 +120,7 @@ export class CreateAccountModel {
       })
       track('Create Account')
     } catch (e: any) {
-      sessionReadySub.remove()
+      this.rootStore.onboarding.skip() // whoops jk
```

Comment:
is this supposed to be here?

Reply:
hah yes, I was being a bit goofy with my comment. I'll fix that